modules/admin_manual/pages/configuration/server/virus-scanner-support.adoc

```diff
@@ -248,7 +248,12 @@ image:configuration/server/anti-virus-message-socket-connection-problem.png[Conf
 
 === What To Do With Infected Files Found During Scans
 
-When infected files are detected during a scan, the app can take one of two actions:
+During an upload both of these actions are taken:
```

Comment:
> During an upload ...
...The upload is blocked

This does not make sense

Reply:
This is not a problem in my humble logic. An upload that gets blocked at some point is a failed or rejected upload. 
Wording alternatives: "the uploaded file gets deleted", "the upload will be rejected", or the "upload will fail" Would that resolve the logic?

We can also phrase it completely differently. The idea that I want to express is: In one case A **and** B happen automatically, in the other case the user preselects A **or** B.

Reply:
This sentence seems OK to me. I rebased the PR, and have expanded the list of things that are done, so that the word "both" does not keep appearing.